modules/@apostrophecms/attachment/index.js

```diff
@@ -1114,7 +1114,7 @@ module.exports = {
               await copyOut(uploadfsPath, tempFile);
               await self.sanitizeSvg(tempFile);
               await copyIn(tempFile, uploadfsPath);
-              await self.db.update({
+              await self.db.updateOne({
```

Comment:
Unrelated: cleaned up legacy mongodb method names that produce warnings. Extremely uninteresting, but it's nice to kill warnings.